public/app/features/alerting/unified/components/rule-editor/alert-rule-form/simplifiedRouting/contactPoint/ContactPointSelector.tsx

```diff
@@ -172,15 +194,16 @@ const getStyles = (theme: GrafanaTheme2) => ({
   }),
   loading: css({
     pointerEvents: 'none',
-    animation: 'rotation 2s infinite linear',
-    '@keyframes rotation': {
-      from: {
-        transform: 'rotate(720deg)',
-      },
-      to: {
-        transform: 'rotate(0deg)',
+    ...handleReducedMotion(
+      {
+        animation: `${rotation} 2s infinite linear`,
```

Comment:
if possible we prefer to keep the rotation here, if disabled it may look the button isn't doing anything at all. I'm far from an expert on the subject though – do smaller items like this trigger vestibular motion disorder?

Reply:
i think any translation/rotation has the potential to trigger, yes. smaller animations are definitely less bad though

in this case the rotation has been replaced by a fade-in/out pulse, so i think it's unlikely anyone would think it's not doing anything. 

https://www.smashingmagazine.com/2023/11/creating-accessible-ui-animations/#3-essential-animations-that-can-be-adapted is a similar scenario, so i wonder if we can reintroduce the rotation at a slower rate along with the pulse. wdyt? 🤔